### PR TITLE
Hypervene Recipe - Part of Big Chem Update

### PR DESCRIPTION
**Pull request by Surrealaser**
Added recipe for Hypervene, a purge chem I've submitted a pull request for.

### CODE REVIEW

File: code/modules/reagents/chemistry_reactions/medical.dm [hunk 1]
```diff
@@ -50,6 +50,13 @@
 	required_reagents = list("sugar" = 1, "phosphorus" = 1, "sulfur" = 1,)
 	result_amount = 3
 
+/datum/chemical_reaction/hypervene
+	name = "Hypervene"
+	id = "hypervene"
+	result = "hypervene"
+	required_reagents = list("arithrazine" = 1, "dylovene" = 1, "ethylredoxrazine" = 1,)
```
Comment (F-OS) [inline code/modules/reagents/chemistry_reactions/medical.dm]: stray comma.

Comment (JamesDenholm) [reply code/modules/reagents/chemistry_reactions/medical.dm]: Stray commas don't actually impact list contents

Comment (F-OS) [reply code/modules/reagents/chemistry_reactions/medical.dm]: they impact my feelings @JamesDenholm

Comment (JamesDenholm) [reply code/modules/reagents/chemistry_reactions/medical.dm]: Man if you wanna talk about feelings you should have seen me reeeing it up on discord

**Comment by Surrealaser** [comment]
Related: https://github.com/ColonialMarines-Mirror/ColonialMarines-2018/pull/168

**Pull request closed without merging**